src/plopp/graphics/color_mapper.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+from ..core.limits import find_limits, fix_empty_range
+from ..core.utils import merge_masks
+
+from matplotlib.colors import Normalize, LogNorm, LinearSegmentedColormap
+from matplotlib import cm
+import scipp as sc
+from copy import copy
+import numpy as np
+
+
+def _get_cmap(name, nan_color=None):
+
+    try:
+        cmap = copy(cm.get_cmap(name))
```

Comment:
Same issue as https://github.com/scipp/scipp/pull/2824

Reply:
Yes.

Reply:
I changed it to match scipp